main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/getsentry/sentry-go"
+
+	"github.com/theartofeducation/go-template-repo/app"
+
+	"github.com/gorilla/mux"
+	"github.com/joho/godotenv"
+
+	"github.com/sirupsen/logrus"
+)
+
+// env variables
+var (
+	port string // The HTTP port the server will run on.
+	dsn  string // The Sentry DSN.
+)
+
+func main() {
+	log := logrus.New()
+
+	loadEnvVariables(log)
+
+	args := app.Args{
+		Router: mux.NewRouter(),
+		Log:    log,
+		DNS:    dsn,
+	}
```

Comment:
So, it looks to me like this answers my question above ... looks like we _can_ in fact depend on those arguments always being there.